pkg/aws/client.go

```diff
@@ -423,32 +415,40 @@ func (b *ClientBuilder) Build() (Client, error) {
 	return c, err
 }
 
-func (c *awsClient) GetIAMCredentials() (credentials.Value, error) {
-	return c.awsSession.Config.Credentials.Get()
+func (c *awsClient) GetIAMCredentials() (aws.Credentials, error) {
+	cfg, err := config.LoadDefaultConfig(context.TODO())
```

Comment:
Do we need to load the config? or can we use the one from the `awsClient` struct?

Reply:
yes we can use the one from the struct